### DISCUSSION TITLE
Cursors

### DISCUSSION THREAD

**Discussion by Donza3D** [inkstitch/inkstitch#3373] [Ideas]
Hi All, I'm new here but not new to beta testing (altho it's been a while). 
Would be great for Inkscape to at least have the eraser tool show a representative size (as in Photoshop) Also would be great to be able to make a selection over several objects & delete from keyboard. I'm finding the ERASE tool very clunky to use. Anyone got any use tips to make it less so?

**Comment by kaalleen** [comment]
Best if you direct this request to Inkscape itself as we do not have any influence on the development of the eraser tool.

https://inkscape.org/report
